### ISSUE TITLE
fix: In IOS, after patch get screen text field error

### ISSUE THREAD

**Issue by LaPyaeOo** [shorebirdtech/shorebird#2318]
App ID: 8a7628f0-f3eb-4e42-839a-2e5afccfe8ea

**Description**

For Android app, patch is all correct and all of the functions are work well but in IOS app after patch, I got UI error (Edit: I only fix for logic error associated with token not fix UI). In a login screen there is three text fields and I focus on last field all of the other components are go forward and all are misalign. Why it happen? Please help me. I need to fix quickly because my application is in production state and so many user will upset my app. If it takes so long.

**Steps To Reproduce**

1. ```shorebird patch --platforms=ios --release-version=1.1.3+54```

2.  
```
Fetching apps... (0.2s)^R
✓ Fetching apps (0.4s)
✓ Fetching releases (0.3s)
✓ Fetching xcarchive artifact (0.3s)
✓ Downloading xcarchive (7.7s)
✓ Building patch with Flutter 3.22.2 (e42d0b27d1) (302.0s)
✓ Verifying patch can be applied to release (1.8s)
✓ Extracting release artifact (0.8s)
✓ Linking AOT files (99.7s)
[WARN] shorebird patch was only able to share 61.4% of Dart code with the released app.
This means the patched code may execute slower than expected.
https://docs.shorebird.dev/status#link-percentage-ios

✓ Generating patch diff base (4.1s)

🚀 Ready to publish a new patch!

📱 App: xxxxxx (8a7628f0-f3eb-4e42-839a-2e5afccfe8ea)
📦 Release Version: 1.1.3+54
🕹️  Platform: ios [arm64 (2.11 MB)]
🟢 Track: Production
🔗 Running 61.4% on CPU
🔍 Debug Info: /Users/xxxxx/xxxx/xxxxxx/xxx-xxx-xxx/build/patch-debug.zip

Would you like to continue? (y/N) Yes
✓ Creating patch (1.9s)
✓ Uploading artifacts (4.0s)
✓ Fetching channels (0.3s)
✓ Promoting patch to stable (0.4s)

✅ Published Patch 6!
```

**Screenshots**

I expected as formal login screen with three text field **username**, **phoneNumber**, **password**. But now as result I got below image

![ImportedPhoto 741794659 526109](https://github.com/shorebirdtech/shorebird/assets/39775137/f8b894f6-3c42-4a28-b852-166778241794)

**Comment by bryanoltman** [comment]
Would you mind sharing a picture of what this is supposed to look like?

**Comment by bryanoltman** [comment]
Also, we're noticing that you have 6 patches for this release. Which patch was active when you took this screenshot?

**Comment by LaPyaeOo** [comment]
I hid my bug screen because it was public but did not hide the important
part. But now I can show from here.
Now I release another version (1 path) with shorebird for IOS. This error
did not happen. But I am afraid to patch it again on IOS due to this bug.

Thanks for your help.

-----------------------------

*Scenario*

When I focus on the last text field called *"PIN"*, all of the screen
widgets go upward. As my code structure I wrapped those all fields with
SingleChildScrollView.
This error happened only on the 6th path of IOS and I screenshot that. Not
happen until the 5th path on IOS. In Android, the 6th path is fine.
-----------------------------

*Bug screen*
.[image: ImportedPhoto.741794659.526109.jpeg]
-----------------------------

*Original Screen that I expected screen*
[image: ImportedPhoto.741898415.717537.jpeg]

**Comment by eseidel** [comment]
This may be another manifestation of https://github.com/shorebirdtech/shorebird/issues/2219.

Shorebird patching is never expected to change behavior outside of your intended changes.  We're investigating 2219 now, and hopefully will resolve this issue.

**Comment by eseidel** [comment]
I believe this was https://github.com/shorebirdtech/shorebird/issues/2219, which we just fixed this afternoon.  We'll have a fix out tomorrow morning.  Thank you for reporting!

**Comment by felangel** [comment]
This should be fixed in v1.1.16. Closing for now but please don't hesitate to reach out or file a new issue if you face any issues using the latest version.

**Comment by eseidel** [comment]
To get the fix you will need to make a new `shorebird release` with Flutter 3.22.2 or later and `shorebird` 1.1.16.
